common/src/main/java/tds/common/configuration/JacksonObjectMapperConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package tds.common.configuration;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import org.springframework.boot.autoconfigure.jackson.Jackson2ObjectMapperBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class JacksonObjectMapperConfiguration {
+    @Bean
+    public Jackson2ObjectMapperBuilderCustomizer customizeObjectMapper() {
+        return builder -> {
+            builder.serializationInclusion(JsonInclude.Include.NON_NULL);
+            builder.modules(new GuavaModule(), new JodaModule());
+        };
+    }
```

Comment:
As discussed offline, we decided to timebox moving the dates from long timestamps to a human-readable date format.  Not a deal breaker, but a nice to have.